Cargo.toml

```diff
@@ -83,29 +80,11 @@ members = [
     "gstd",
     "gsys",
     "gtest",
-    "node/cli",
-    "node/service",
-    "node/authorship",
-    "node/testing",
+    "node/[!res]*[!md]",
```

Comment:
I'd leave the old list for `node` as this `[!res]*[!md]` looks not too cool to me.

Reply:
but this is a one-line solution )) mb create a poll : D

Reply:
leave it to #3069